src/Mailer/Ownership/RecipientStrategy/RecipientStrategy.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Mailer\Ownership\RecipientStrategy;
+
+use CultuurNet\UDB3\Ownership\Repositories\OwnershipItem;
+use CultuurNet\UDB3\ReadModel\DocumentDoesNotExist;
+use CultuurNet\UDB3\User\UserIdentityDetails;
+
+interface RecipientStrategy
+{
+    /**
+     * @throws DocumentDoesNotExist
```

Comment:
I'm not sure this throw is generic, because in the `SendToOwnerOfOwnership` implementation it returns an empty array.